src/components/tabs/Tab.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  export let key;
+  import { getContext } from "svelte";
+
+  const store = getContext("activeTab");
```

Comment:
I *think* using `activeTab` as a name here makes more sense (basically we just have one piece of context).

```suggestion
  const activeTab = getContext("activeTab");
```